config/database.go

```diff
@@ -1,27 +1,30 @@
 package config
 
 import (
-	"database/sql"
 	"fmt"
 )
 
 type DatabaseConfig struct {
+	DSN      string
```

Comment:
```suggestion
	dsn      string
```
already have `GetDSN()` method, should keep `DSN` private.

Reply:
I was going to add a new configuration field for the database section. For example:

database:
  dsn: postgres://username:password@127.0.0.1:5432/webhookx?sslmode=disable
  ...
  
The `dsn` will have the highest priority compared the the rest of all (username, password, etc)

Reply:
deleted. (will be done on another PR)